src/lib/components/TokenSelect/index.tsx

```diff
@@ -17,14 +18,18 @@ const SearchInput = styled(StringInput)`
   ${inputCss}
 `
 
-export function TokenSelectDialog({ onSelect }: { onSelect: (token: Currency) => void }) {
+interface TokenSelectDialogProps {
+  value?: Currency
```

Comment:
can this be called `currency`?

Reply:
I want to keep it as value to mirror native select elements, and make it obvious that this is the currently selected value.
I think all the tokens should be replaced with currency, but am waiting to do that until we've got some down time.